words/predicates.txt

```diff
@@ -1105,14 +1244,18 @@ wiry
 wise
 wistful
 witty
+wobbly
 wonderful
 wood
+wooded
 wooden
 wool
+woolen
 woozy
 workable
 worried
 wry
+yellow
```

Comment:
Any concerns about possible racist combinations?

Reply:
probably why it wasn't on there previously, good call.